### PR TITLE
binance: round 2, now with app check

### PR DESCRIPTION
**Pull request by nalinbhardwaj**

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/DepositScreen.tsx [hunk 1]
```diff
@@ -151,18 +152,18 @@ function getOnDemandExchanges(
...
-  const rpcFunc = getRpcFunc(daimoChainFromId(account.homeChainId));
+  const daimoChain = daimoChainFromId(account.homeChainId);
+  const rpcFunc = getRpcFunc(daimoChain);
+  const deviceType = env(daimoChain).deviceType;
 
-  const platform = ["ios", "android"].includes(Platform.OS)
-    ? (Platform.OS as PlatformType)
-    : "other";
+  const platform =
+    deviceType === "computer" ? "other" : (Platform.OS as PlatformType);
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/DepositScreen.tsx:160]: 👍 
this is tricky logic, might even put it in env so we can say

```ts
const { platform } = env(daimoChain);
```

File: apps/daimo-mobile/src/view/screen/DepositScreen.tsx [hunk 2]
```diff
@@ -176,15 +177,33 @@ function getOnDemandExchanges(
...
+          "Please install Binance to use this feature",
+          [
+            {
+              text: "Continue",
+              onPress: async () => {
+                await Linking.openURL(url);
+              },
+            },
+            { text: "Cancel" },
+          ]
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/DepositScreen.tsx:194]: 👍

**Review by dcposch — APPROVED** [review]
LGreatTM

**Comment by nalinbhardwaj** [comment]
closing for now

**Pull request closed without merging**